index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Documentation Page</title>
+</head>
+<body>
+  <div class="container">
+    <div class="sideNav">

```

Comment:
- [ ] I suggest you explain why this change is required, that will make it very easy to implement the changes because the one who will read it will know the context

Reply:
Please use an aside tag instead of a div because:
* The aside element defines some content aside from the content it is placed in (like a sidebar).
* The aside content should be indirectly related to the surrounding content.